samples/getting-started/azure-quantum/README.md

```diff
@@ -0,0 +1,61 @@
+---
+page_type: sample
+languages:
+- python
+products:
+- qdk
+- azure-quantum
+description: "This sample demonstrates how to use Honeywell and IonQ on the Azure Quantum service with quantum circuits expressed in various Python libraries."
```

Comment:
This description and the following URL fragment make it sound like using third-party libraries is the only way to use the Azure Quantum service with Python?

Reply:
It's more of a statement on how to use the Azure Quantum service with _quantum circuits expressed in Python libraries_.

Reply:
I have split up the sample into sub folders so I believe this issue has been addressed.